code/modules/client/preference_setup/loadout/loadout_eyes.dm

```diff
@@ -53,7 +53,7 @@
 /datum/gear/eyes/medical
 	display_name = "Medical HUD (Medical)"
 	path = /obj/item/clothing/glasses/hud/health
-	allowed_roles = list("Medical Doctor","Chief Medical Officer","Chemist","Paramedic","Geneticist")
+	allowed_roles = list("Medical Doctor","Chief Medical Officer","Chemist","Paramedic","Geneticist", "Psychiatrist")
```

Comment:
I am not particularly a fan of this change.

Reply:
If chemists and geneticists (neither of which 'need' one per se) can have it through the loadout, and their job has access to it...

Reply:
Chemists have a good reason to have a MedHud, as much as I might personally dislike that. Psychiatrists really, really do not.